src/main.rs

```diff
@@ -369,7 +369,7 @@ fn kernel_main(boot_info: &'static BootInfo) -> ! {
     if let Some(mm) = mm_lock.as_mut() {
        match Process::new(mm) {
            Ok(mut init_process) => {
-               init_process.current_dir = String::from("/");
+               init_process.current_dir = "/".to_string();
```

Comment:
Sorry, this is minor, but Rust analyzer was complaining about `String::from("/")`.